### PR TITLE
new make command to run a remote branch via a prebuilt docker image

### PR DESCRIPTION
**Pull request by sckott**
Added a few things to run 
- `--rm`: cleanup
- `-it`: acknowledge `ctrl+c` so container can be killed from the cli

### CODE REVIEW

**Comment by sckott** [comment]
Oh, we should add how to get credentials. What should that say Dan?

**Review by dtenenba — APPROVED** [review]
Looks good!

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ Also note that docker commands can be specific to the host operating system, so
 The make command `make run_branch` will run a remote branch locally on your machine. Whereas `make run_docker` runs the current branch you're on, `make run_branch` pulls down a docker image of the remote branch specified and runs that. 
 
 Make sure to run `docker login nexus-registry.fredhutch.org` before you try this make command. You only need to run this once.
+Contact `scicomp` to get the username and password for this step.
```
Comment (sckott) [inline README.md:46]: great, thanks

**Comment by sckott** [comment]
Thanks for having a look Dan

**Pull request merged**